src/components/application_manager/src/application_data_impl.cc

```diff
@@ -572,6 +573,16 @@ void DynamicApplicationDataImpl::RemoveChoiceSet(uint32_t choice_set_id) {
     delete it->second;
     choice_set_map_.erase(choice_set_id);
   }
+
+  ChoiceSetAllowedMap::iterator choise_id_it =
+      choice_set_allowed_map_.find(choice_set_id);
+  if (choice_set_allowed_map_.end() == choise_id_it) {
+    LOG4CXX_WARN(logger_,
+                 "Not exists info about choice set " << choice_set_id
```

Comment:
@IGapchuk "Choice set with id " << choice_set_id << " is not found"

Reply:
@AKalinich-Luxoft Fixed in 466447c82ad8882ea67b47f96fcc10788db6351f.